[2025-06-05T16:07:31.177+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.delete_data_from_table manual__2025-06-05T16:04:11.870740+00:00 [queued]>
[2025-06-05T16:07:31.193+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.delete_data_from_table manual__2025-06-05T16:04:11.870740+00:00 [queued]>
[2025-06-05T16:07:31.194+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 7
[2025-06-05T16:07:31.213+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2025-06-05 16:04:11.870740+00:00
[2025-06-05T16:07:31.218+0000] {standard_task_runner.py:60} INFO - Started process 54305 to run task
[2025-06-05T16:07:31.222+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'delete_data_from_table', 'manual__2025-06-05T16:04:11.870740+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/postgresql_dag.py', '--cfg-path', '/tmp/tmps6b258ki']
[2025-06-05T16:07:31.224+0000] {standard_task_runner.py:88} INFO - Job 100: Subtask delete_data_from_table
[2025-06-05T16:07:31.356+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.delete_data_from_table manual__2025-06-05T16:04:11.870740+00:00 [running]> on host 14aa2e8b0b7d
[2025-06-05T16:07:31.498+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v01' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2025-06-05T16:04:11.870740+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-05T16:04:11.870740+00:00'
[2025-06-05T16:07:31.502+0000] {sql.py:276} INFO - Executing: 
            delete from dag_runs where dt = '2025-06-05' and dag_id = 'dag_with_postgres_operator_v01';
        
[2025-06-05T16:07:31.517+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-05T16:07:31.541+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-05T16:07:31.588+0000] {sql.py:450} INFO - Running statement: 
            delete from dag_runs where dt = '2025-06-05' and dag_id = 'dag_with_postgres_operator_v01';
        , parameters: None
[2025-06-05T16:07:31.599+0000] {sql.py:459} INFO - Rows affected: 1
[2025-06-05T16:07:31.622+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=delete_data_from_table, execution_date=20250605T160411, start_date=20250605T160731, end_date=20250605T160731
[2025-06-05T16:07:31.681+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-05T16:07:31.710+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
